src/ripple/protocol/impl/Serializer.cpp

```diff
@@ -470,29 +445,35 @@ bool Serializer::getVLLength (int& length, int offset) const
     return true;
 }
 
-Blob Serializer::encodeVL (int length)
+int Serializer::addEncoded (int length)
 {
-    unsigned char lenBytes[4];
+    std::array<std::uint8_t, 4> result;
+    int l = encodeVL (length, result);
```

Comment:
`l` is generally a poor choice for variable names because of its similarity to `1` and `I`.

Why not just inline it into the next line?